contracts/ICS20Transfer.sol

```diff
@@ -233,7 +228,7 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
         string calldata destChannel,
         ICS20Lib.FungibleTokenPacketData memory packetData
     )
-        private
+        public
```

Comment:
Isn't this a huge security issue given that this function is not view (state modifying)

Reply:
Yeah, it probably should not be public.

Reply:
Linking [this point](https://github.com/cosmos/solidity-ibc-eureka/pull/126#discussion_r1869007496) here

Reply:
Changed back to private. Instead I added a public view getter function of the _ibcDenomContracts private variable.